src/client/game.cpp

```diff
@@ -3070,10 +3070,9 @@ void Game::processPlayerInteraction(f32 dtime, bool show_hud, bool show_debug)
 
 #endif
 
-	PointedThing pointed = updatePointedThing(shootline,
-			selected_def.liquids_pointable,
-			!runData.btn_down_for_dig,
-			camera_offset);
+	PointedThing pointed;
+	if (d != -1) pointed = updatePointedThing(shootline, selected_def.liquids_pointable,
+			!runData.down_for_dig, camera_offset);
```

Comment:
```suggestion
	if (d != -1)
		pointed = updatePointedThing(shootline,
				selected_def.liquids_pointable,
				!runData.btn_down_for_dig,
				camera_offset);
```
Compilation error due to `down_for_dig` not being a member of `GameRunData`
And code style